netbox/dcim/tests/test_filtersets.py

```diff
@@ -593,11 +593,11 @@ def setUpTestData(cls):
         Rack.objects.bulk_create(racks)
 
         users = (
-            User(username='User 1'),
-            User(username='User 2'),
-            User(username='User 3'),
+            get_user_model()(username='User 1'),
```

Comment:
I'm sure there's negligible overhead from calling `get_user_model()` multiple times, but IMO it would be a bit cleaner to just do `User = get_user_model()` at the top of the file and reference `User` normally. This would also minimize the number of changes necessary.